packages/core/src/internal/utils/validators.ts

```diff
@@ -119,8 +119,16 @@ export function validateTopK(topK: number) {
 }
 
 export function validateTtlMinutes(ttlMinutes: number) {
-  if (ttlMinutes < 0) {
-    throw new InvalidArgumentError('ttlMinutes must be positive');
+  if (ttlMinutes < 0 || !Number.isInteger(ttlMinutes)) {
```

Comment:
I think we missed this for `safeInteger`.. dont want to nitpick but just to be consistent

Reply:
oh yes, good catch, thanks!